src/angular/app/leagues/league-profile/league-profile.component.ts

```diff
@@ -77,12 +77,20 @@ export class LeagueProfileComponent
     }
 
     ngAfterViewInit(): void {
-        setTimeout(() => this.initFloatingButtons(), 500);
+        // Poll until loaded
+        const buttonPollId = setInterval(() => {
+            if (this.league) {
+                this.initFloatingButtons();
+                clearInterval(buttonPollId);
+            }
+        }, 500);
     }
 
     initFloatingButtons(): void {
         const buttons = document.querySelectorAll('.fixed-action-btn');
-        M.FloatingActionButton.init(buttons);
+        console.log(buttons);
```

Comment:
Is this on purpose or you forgot to remove it?

Reply:
Think i just forgot